src/themes/types.ts

```diff
@@ -30,11 +106,19 @@ export type State = ObjectOf<any>
 // ========================================================
 
 export interface SiteVariablesInput extends ObjectOf<any> {
+  colors?: ColorPalette
+  contextualColors?: ContextualColors
+  emphasisColors?: EmphasisColors
+  naturalColors?: NaturalColorsStrict
   brand?: string
```

Comment:
what this key stands for? is it theme's name?

Reply:
Defines brand color in our themes, should be definitely removed in next PRs.